packages/frontend/src/components/Dataset/Dataset.scss

```diff
@@ -1,6 +1,11 @@
 .dataset {
   display: grid;
 
+  .data-link-title{
+    &:focus-visible{
```

Comment:
Remember to keep your code following any patterns you already see in the codebase you're editing. In this case, remember to add a space before the opening curly braces.